### PR TITLE
[IMP] Base: Better handling of indexes

### PR DESCRIPTION
**Pull request by VincentSchippefilt**
Introduct changelog of the ORM by version, link to first change documented

### CODE REVIEW

**Comment by VincentSchippefilt** [comment]
linked to https://github.com/odoo/odoo/pull/83015 and https://github.com/odoo/odoo/pull/83274

File: content/developer/reference/backend/orm_changelog.rst [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+=============
+
+
+version 15.2
+============
+
+specific index types on fields
+------------------------------
+With https://github.com/odoo/odoo/pull/83274 and https://github.com/odoo/odoo/pull/83015 developers can now define what type of indexes can be used by postgresql on fields
+:ref:`see the index property of Field <reference/fields>`.
```
Comment (xmo-odoo) [inline content/developer/reference/backend/orm_changelog.rst:15]: Assuming this is in large parts for external consumption, it should probably have 15.1 as well?

**Comment by VincentSchippefilt** [comment]
@AntoineVDV @xmo-odoo thanks for the reviews. Indeed it is better with denser changes :-)

File: content/developer/reference/backend/orm_changelog.rst [hunk 2]
```diff
@@ -0,0 +1,15 @@
+
+.. _reference/orm_changelog:
+
+=============
+ORM Changelog
+=============
+
+
+version 15.2
```
[the remark this answers is not in the document]
Comment (VincentSchippefilt) [reply content/developer/reference/backend/orm_changelog.rst]: is "Odoo Online" a well known name for the saas ?

File: content/developer/reference/backend/orm_changelog.rst (hunk 1 above)
Comment (xmo-odoo) [reply content/developer/reference/backend/orm_changelog.rst]: No, I mean the changelog for saas-15.1 should be included.

File: content/developer/reference/backend/orm_changelog.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (AntoineVDV) [reply content/developer/reference/backend/orm_changelog.rst]: [Yes](https://www.odoo.com/documentation/master/administration/maintain/hosting_changes.html).

**Review by ryv-odoo — APPROVED** [review]
@robodoo r+

**Comment by VincentSchippefilt** [comment]
@robodoo r+ please

**Comment by VincentSchippefilt** [comment]
@AntoineVDV can you r+ ?

**Comment by AntoineVDV** [comment]
Force-pushed to fix English issues and add some minor improvements.

**Comment by AntoineVDV** [comment]
@robodoo r+

**Pull request closed without merging**